2020/11/09 16:04:36 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/09 16:15:38 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/09 16:15:39 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085", referrer: "http://localhost:8085/"
2020/11/09 16:15:40 [error] 27#27: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/09 16:31:00 [error] 27#27: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/09 16:31:00 [error] 27#27: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085", referrer: "http://localhost:8085/"
2020/11/10 14:32:52 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 14:32:52 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085", referrer: "http://localhost:8085/"
2020/11/10 14:32:53 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 14:32:54 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 14:32:54 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 14:32:55 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 14:32:56 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 14:33:09 [error] 28#28: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 14:33:53 [error] 28#28: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "127.0.0.1:8085"
2020/11/10 14:33:53 [error] 28#28: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "127.0.0.1:8085", referrer: "http://127.0.0.1:8085/"
2020/11/10 14:35:26 [error] 28#28: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "127.0.0.1:8085"
2020/11/10 14:35:26 [error] 28#28: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "127.0.0.1:8085", referrer: "http://127.0.0.1:8085/"
2020/11/10 15:38:33 [crit] 27#27: *1 realpath() "/etc/nginx/html" failed (2: No such file or directory), client: 172.19.0.1, server: ~.*, request: "GET / HTTP/1.1", host: "localhost:8085"
2020/11/10 15:38:33 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: ~.*, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.2:9500", host: "localhost:8085"
2020/11/10 15:41:06 [crit] 27#27: *1 realpath() "/etc/nginx/html" failed (2: No such file or directory), client: 172.19.0.1, server: ~.*, request: "GET / HTTP/1.1", host: "localhost:8085"
2020/11/10 15:41:06 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: ~.*, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 15:41:06 [crit] 27#27: *1 realpath() "/etc/nginx/html" failed (2: No such file or directory), client: 172.19.0.1, server: ~.*, request: "GET / HTTP/1.1", host: "localhost:8085"
2020/11/10 15:41:06 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: ~.*, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 15:41:07 [crit] 27#27: *1 realpath() "/etc/nginx/html" failed (2: No such file or directory), client: 172.19.0.1, server: ~.*, request: "GET / HTTP/1.1", host: "localhost:8085"
2020/11/10 15:41:07 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: ~.*, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 15:41:07 [crit] 27#27: *1 realpath() "/etc/nginx/html" failed (2: No such file or directory), client: 172.19.0.1, server: ~.*, request: "GET / HTTP/1.1", host: "localhost:8085"
2020/11/10 15:41:07 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: ~.*, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 15:41:08 [crit] 27#27: *1 realpath() "/etc/nginx/html" failed (2: No such file or directory), client: 172.19.0.1, server: ~.*, request: "GET / HTTP/1.1", host: "localhost:8085"
2020/11/10 15:41:08 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: ~.*, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 16:01:53 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 16:01:54 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 16:01:54 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 16:01:54 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 16:01:55 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 16:01:55 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 16:01:55 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 16:01:55 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 16:01:55 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 16:01:56 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9500", host: "localhost:8085"
2020/11/10 16:07:05 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.2:9500", host: "localhost:8085"
2020/11/10 16:07:05 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.19.0.2:9500", host: "localhost:8085", referrer: "http://localhost:8085/"
2020/11/10 16:07:05 [error] 27#27: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.2:9500", host: "localhost:8085"
2020/11/10 16:11:35 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: [info] User Deprecated: Using the WebserverBundle is deprecated since Symfony 4.4. The new Symfony local server has more features, you can use it instead.
PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /var/www/practice/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.2:9000", host: "localhost:8085"
2020/11/11 12:59:39 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: [info] User Deprecated: Using the WebserverBundle is deprecated since Symfony 4.4. The new Symfony local server has more features, you can use it instead.
PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /var/www/practice/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.19.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.2:9000", host: "localhost:8085"
